### ISSUE TITLE
xattr=sa support?

### ISSUE THREAD

**Issue by Ambyjkl** [openzfsonwindows/ZFSin#336]
https://github.com/openzfsonwindows/ZFSin/pull/100 says xattr=sa is not implemented yet. Is it still the case?

**Comment by lundman** [comment]
That is correct - it wouldn't be hard to add, but since it is mostly a performance thing, it's been lower priority.

**Comment by Ambyjkl** [comment]
Yeah, for sure, things like the OpenZFS 2.0 rebase are definitely more important. Looking forward to when this gets implemented. Also wanted to point out that https://openzfs.org/wiki/Platform_code_differences places `xattr=sa` in the "platform independent" section, which is kinda misleading.
